.github/workflows/release.yml

```diff
@@ -99,6 +101,15 @@ jobs:
         asset_path: ${{ github.workspace }}/lpk-win-x64/lpk-win-x64.zip
         asset_name: lpk-win.zip
         asset_content_type: application/zip
+    - name: Upload Window ARM64 Release Assets
```

Comment:
Could we replace all six of these upload steps with the following?
```yaml
    - name: Upload Release Assets
      uses: actions/upload-release-asset@v1 
      env:
        GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
      with:
        upload_url: ${{ steps.create_release.outputs.upload_url }}
        asset_path: ${{ github.workspace }}/${{ matrix.assetBuildName }}/${{ matrix.assetBuildName }}.zip
        asset_name: ${{ matrix.assetName }}.zip
        asset_content_type: application/zip
```

And in the matrix `includes` section:
```yaml
- mode: osx-x64
  ...
  assetBuildName: lpk-osx-x64
  assetName: lpk-osx
- mode: linux-x64
  ...
  assetBuildName: lpk-linux-x64
  assetName: lpk-linux
- mode: win-x64
  ...
  assetBuildName: lpk-win-x64
  assetName: lpk-win
- mode: win-arm64
  ...
  assetBuildName: lpk-win-arm64
  assetName: lpk-win-arm64
- mode: osx-arm64
  ...
  assetBuildName: lpk-osx-arm64
  assetName: lpk-osx-arm64
- mode: linux-arm64
  ...
  assetBuildName: lpk-linux-arm64
  assetName: lpk-linux-arm64
```

Reply:
@peterbom Updated, thank you.